config/policies.js

```diff
@@ -75,6 +75,9 @@ module.exports.policies = {
   ],
   'game/seven/untargeted-one-off': ['isLoggedIn', 'isInGame', 'hasCardId'],
 
+  //GameStateApi
+  'game/:gameid/move': ['isLoggedIn', 'isInGame', 'hasMoveType'],
```

Comment:
I think we should remove the isInGame policy here in favor of a check in the move making endpoint.  


The main reason is because I think the check in being in the game should be a db read on the game obj matching your user id in p0 or p1. (Currently it's just a check if your redis session, which is easier to desync and is less conducive to being in multiple games). 

We already query the game in the endpoint so it is inefficient to query for it in the policy as well. An alternative could be to do the check in the policy but make that the place where we actually query for the game, appending the locked game into the `req` object so the endpoint can access it as `req.game`. That way we still only query once but the logic for verifying access is kept in the policy which is arguably more reusable.

It would likely be more concise but also arguably more magical since the endpoint would assume a query had been made in the policy that you don't see directly in the endpoint code. That makes me lean towards just checking in the endpoint itself